.github/workflows/gitversion.yml

```
@@ -29,10 +29,10 @@ jobs:
       run: |
         git config --global --add safe.directory /github/workspace
 
-    - name: Setup .NET 6
+    - name: Setup .NET 8
       uses: actions/setup-dotnet@v2
```

actionlint (1.7.7)

the runner of "actions/setup-dotnet@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)